kahuna/public/js/components/gr-top-bar/gr-top-bar.js

```diff
@@ -22,8 +22,7 @@ topBar.directive('grTopBarNav', [function() {
         transclude: true,
         // Annoying to have to hardcode root route here, but only
         // way I found to clear $stateParams from uiRouter...
-        template: `<a href="/search" class="home-link">Home</a>
-                   <ng:transclude></ng:transclude>`
+        template: `<ng:transclude></ng:transclude>`
```

Comment:
Moved to the html template rather than in js.

See `kahuna/public/js/search/view.html:3`